src/startupscriptgenerator/src/restorePackages.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
1. add copyright notice
2. consider renaming file to match our more common convention (`restore-packages.sh`)